components/__tests__/__snapshots__/locale-chooser.js.snap

```diff
@@ -274,21 +265,25 @@ exports[`opened state 1`] = `
           >
             <glamorous(li)>
               <li
-                className="css-l0f4u7"
+                className="css-1uz022"
               >
                 <glamorous(a)
-                  href="about://blank"
+                  aria-label="Change localization to English"
+                  href="about://undefinedblank"
                 >
                   <a
```

Comment:
Does this go `a[aria-label="Change localization to English"] > div > svg[alt="English"]`? If so, I'd recommend changing the the `alt` attribute to be blank on the SVGs here so they're considered decorative.